### PR TITLE
Add app images

### PR DESCRIPTION
**Pull request by DanielSporysz**

### CODE REVIEW

**Review by PatrykZaniewski — DISMISSED** [review]

File: appstore/utils/validator/file_validator.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+from fastapi import UploadFile
+
+from appstore.exceptions.appstore_exceptions import InvalidFile
+
+
+def file_is_valid(file: UploadFile) -> bool:
+    if file is None:
+        raise InvalidFile
+
+    if file.content_type.capitalize() != "image/jpeg":
```
Comment (KonradMagiera) [inline appstore/utils/validator/file_validator.py]: Do you allow only jpeg files?

Comment (maciekkoz98) [inline appstore/utils/validator/file_validator.py]: only jpeg? at least add png

**Review by maciekkoz98 — CHANGES_REQUESTED** [review]

**Review by KonradMagiera — CHANGES_REQUESTED** [review]

File: appstore/utils/validator/file_validator.py (hunk 1 above)
Comment (DanielSporysz) [reply appstore/utils/validator/file_validator.py]: Our database does not have a field for filename nor file type. Please escalate the issue to our DevOps team.

Comment (maciekkoz98) [reply appstore/utils/validator/file_validator.py]: that's your problem

Comment (DanielSporysz) [reply appstore/utils/validator/file_validator.py]: Our database does not have a field for filename nor file type. Please escalate the issue to our DevOps team.

Comment (DanielSporysz) [reply appstore/utils/validator/file_validator.py]: The same request has been already submitted. I'm closing this one.

Comment (DanielSporysz) [reply appstore/utils/validator/file_validator.py]: Support for png filles will be added as another feature in future commits.

**Review by KatarzynaWolska — APPROVED** [review]

**Review by KonradMagiera — APPROVED** [review]

**Review by PatrykZaniewski — APPROVED** [review]

**Review by maciekkoz98 — APPROVED** [review]

**Pull request merged**